### DISCUSSION TITLE
Adding containers to gluetun breaks existing networking of these containers (without network_mode: "service:gluetun" networking is working)

### DISCUSSION THREAD

**Discussion by distributev** [qdm12/gluetun#2413] [Q&A]
Hello,
I spent a lot of time on this I believe it is a bug but maybe not.
I went through many similar issues, I tried all possible configs but now luck.

Here are all the details.

https://github.com/qdm12/gluetun/issues/2412

**Comment by bluskye** [comment]
Did you open the ports?


services:
  gluetun:
    image: qmcgaw/gluetun:latest
    container_name: gluetun
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun:/dev/net/tun
    ports:
      - 8888:8888/tcp # HTTP proxy
      - 8112:8112 #deluge
      - 6881:6881 #deluge
      - 6881:6881/udp #deluge
      - 9091:9091 #transmission
      - 51413:51413 #transmission
      - 51413:51413/udp #transmission
